runtime/polkadot/src/lib.rs

```diff
@@ -1427,6 +1427,8 @@ pub type Migrations = (
 	// Unreleased - add new migrations here:
 	pallet_nomination_pools::migration::v5::MigrateToV5<Runtime>,
 	parachains_configuration::migration::v5::MigrateToV5<Runtime>,
+	/* Asynchronous backing mirgration */
```

Comment:
nit: consistent comment syntax
```suggestion
	// Asynchronous backing mirgration
```

Reply:
This is intentional. Similar to https://github.com/paritytech/polkadot/blob/fa0ad9ab36cff1d124be97c02ebaebe521c3781e/primitives/src/runtime_api.rs#L222-L231

Note that this PR targets feature branch, not master. Our migrations shouldn't be dropped unless applied on all networks, unlike migrations coming from master.